src/policy.rs

```diff
@@ -30,36 +32,46 @@ impl PolicyAxis {
         }
     }
 
-    #[allow(clippy::len_without_is_empty)]
+    /// Returns the number of attributes belonging to this axis.
     #[must_use]
     pub fn len(&self) -> usize {
         self.attributes.len()
     }
 
+    pub fn is_empty(&self) -> bool {
+        self.len() == 0
+    }
+
+    /// Returns the list of attribute names belonging to this axis.
     pub fn attributes(&self) -> &[String] {
         &self.attributes
     }
 
+    /// Returns the axis name.
     pub fn name(&self) -> &str {
         &self.name
     }
 
+    /// Returns `true` if this axis is hierarchical.
     pub fn is_hierarchical(&self) -> bool {
         self.hierarchical
     }
 }
 
-/// A policy is a set of fixed policy axes, defining an inner attribute
-/// element for each policy axis attribute a fixed number of revocation
-/// addition of attributes is allowed
+/// A policy is a set of policy axes. A fixed number of attribute creations
+/// (revocations + additions) is allowed.
 #[derive(Clone, Serialize, Deserialize, Debug)]
 pub struct Policy {
-    pub last_attribute_value: u32,
-    pub max_attribute_value: u32,
-    /// store the policies by name
-    pub store: HashMap<String, (Vec<String>, bool)>,
+    /// Last value taken by the attriute.
+    last_attribute_value: u32,
+    /// Maximum attribute value. Defines a maximum number of attribute
+    /// creations (revocations + addition).
+    pub max_attribute_creations: u32,
```

Comment:
it is a breaking change. Renaming this impacts the serialization: the TTLV serialization in KMS must be updated, and so the cosmian_java_lib. Also it impacts the WASM and hence the cosmian_js_lib. 
Is it what we want?

Reply:
the version will reflect the breaking change. Upgrading is not required absolutely.